hate-detect/common_ngrams_finder.py

```diff
@@ -1,10 +1,13 @@
 import nltk
+#nltk.download('words')
+#nltk.download('averaged_perceptron_tagger')
+#nltk.download('universal_tagset')
```

Comment:
Where should we put these?

Reply:
I think we should move them to a config file or something?